### ISSUE TITLE
Error on Intel Atom: Illegal instruction

### ISSUE THREAD

**Issue by pkolt** [lovell/sharp#2809]
Hi! I have an error "Illegal instruction" when working with a sharp.

This issue arose from a discussion of bug ["Does not work strapi start since v3.6"](https://github.com/strapi/strapi/issues/10146).

**What are the steps to reproduce?**

1. `git clone git@github.com:pkolt/sharp_for_strapi.git`
2. `npm ci`
3. `npm test`
4. Error in console "Illegal instruction"

The error is repeated on the Intel Atom architecture.

**What is the expected behaviour?**

The code runs without errors.

**Are you able to provide a minimal, standalone code sample, without other dependencies, that demonstrates this problem?**

```js
const sharp = require('sharp');

sharp('example.jpg')
    .resize(300, 200)
    .toFile('out.jpg', (err) => {
        console.error(err);
    });
```

**Are you able to provide a sample image that helps explain the problem?**

```
System:
  OS: Linux 4.19 Debian GNU/Linux 10 (buster) 10 (buster)
  CPU: (2) x64 Intel(R) Atom(TM) CPU D525   @ 1.80GHz
  Memory: 224.09 MB / 3.85 GB
  Container: Yes
  Shell: 5.0.3 - /bin/bash
Binaries:
  Node: 14.16.1 - ~/.nvm/versions/node/v14.16.1/bin/node
  npm: 6.14.12 - ~/.nvm/versions/node/v14.16.1/bin/npm
```

**Comment by lovell** [comment]
Please see #2723

**Comment by pkolt** [comment]
@lovell Thanks! My problem solved with sharp@0.27.2

**Comment by Isti115** [comment]
Just for future reference, in case anyone's looking to override the version of sharp used in their projects, but have it as a nested dependency, in case of `npm` you can use the `resolutions` key of `package.json` like so:
```js
[...]
  "resolutions": {
    "sharp": "0.27.2"
  },
[...]
```
